controllers.py

```diff
@@ -0,0 +1,38 @@
+import models
+import views
+import pendulum
+
+
+def validate_new_task(task):
+    pass
+
+
+def add_new_task(task=None):
```

Comment:
Not an issue at all for your application but usually we would group controller by domain, so I was expecting to see a TaskController with create/read/update/delete/search/etc.. methods.